src/main/resources/intentionDescriptions/JoinWildcardsIntention/description.html

```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+This intention converts successive <code>_</code> wildcard(s) to <code>..</code>
```

Comment:
```suggestion
Converts successive <code>_</code> wildcard(s) to <code>..</code>
```

Reply:
Almost all our intention descriptions start with `This intention ...`. I think we should rewrite them all at once